src/cmdline.rs

```diff
@@ -654,6 +715,29 @@ pub struct PxeIgnitionUnwrapConfig {
     pub input: Option<String>,
 }
 
+#[derive(Debug, StructOpt)]
+pub struct PxeNetworkWrapConfig {
+    /// NetworkManager keyfile to embed
+    // Required option. :-(  In future we might support other configuration
+    // sources.
+    #[structopt(short, long, required = true, value_name = "path")]
+    #[structopt(number_of_values = 1)]
+    pub keyfile: Vec<String>,
+    /// Write to a file instead of stdout
+    #[structopt(short, long, value_name = "path")]
+    pub output: Option<String>,
+}
+
+#[derive(Debug, StructOpt)]
+pub struct PxeNetworkUnwrapConfig {
+    /// Destination directory
+    #[structopt(short = "C", long, value_name = "path")]
+    pub directory: Option<String>,
```

Comment:
It's slightly non-intuitive I think that not specifying a directory is not equivalent to specifying the cwd, but I don't have a better idea.

Reply:
Yeah, agreed.  Clarified the help text a bit.